package.json

```diff
@@ -72,7 +72,7 @@
       }
     },
     "test": {
-      "command": "mocha-webpack --webpack-config webpack/config.test.js --recursive \"src/**/__tests__/*.js\" --full-trace --require source-map-support/register --colors",
+      "command": "mocha-webpack --webpack-config webpack/config.test.js --recursive \"src/**/__tests__/*.js\" --full-trace --require source-map-support/register --require ignore-styles --colors",
```

Comment:
Explanation for this change: even with 'ignore-loader' being set in config.test.js I was getting "Unexpected token" errors from css files which were being loaded from node_modules. Adding 'ignore-styles' ensures that all these css files are ignored as well, which gets rid of the errors. (We still need 'ignore-loader' in config.test.js, though).